robyn/router.py

```diff
@@ -88,16 +87,16 @@ def _format_response(
             if len(res) != 3:
                 raise ValueError("Tuple should have 3 elements")
             else:
-                description, headers, status_code = res
-                description = self._format_response(description).description
-                new_headers = Headers(headers)
-                if "Content-Type" in new_headers:
-                    headers.set("Content-Type", new_headers.get("Content-Type"))
+                description2, headers2, status_code2 = res
+                description2 = self._format_response(description2).description
+                new_headers: Headers = Headers(headers2)
+                if new_headers.contains("Content-Type"):
+                    headers2.set("Content-Type", new_headers.get("Content-Type"))
```

Comment:
Hey @dave42w 👋 

Thank you for the PR :D
One question - 
why are we using `description2` or `headers2` here? Is it something that mypy suggests?

Reply:
They just have to be different names to the ones earlier in _format_response. mypy objects to reusing a variable (even if it has gone out of scope) with a different type. eg this is unacceptable

```
if x:
  y: int = 5
else:
  y: str = "five"
```

Reply:
@sansyrox are we ok to consider this point resolved?

Reply:
Hey @dave42w 👋 

Apologies for the late review.

> They just have to be different names to the ones earlier in _format_response. mypy objects to reusing a variable (even if it has gone out of scope) with a different type. eg this is unacceptable

We can disable mypy for these lines then. 

> mypy objects to reusing a variable (even if it has gone out of scope) with a different type

This is not a good reason IMO. If the code makes sense in our repo, we should not alter it according to mypy.

Reply:
H'mm, it seems risky to me. We are deep in lots of nesting at this point. It would be easy for a code change to end up trying to use these variables later in the function (especially because we don't return early) and depending on the data have 2 different types.
I would like to follow this with a more significant refactor anyway to reduce the levels of indentation/nested if statements. At that point this will not be a problem.

Reply:
Alright then. but can we get rid of numbers in the variable names. I get a massive ick by that(I know and I am sorry) but something else without involving numbers?